babel/core.py

```diff
@@ -885,6 +885,41 @@ def ordinal_form(self):
         """
         return self._data.get('ordinal_form', _default_plural_rule)
 
+    @property
+    def measurement_systems(self):
+        """Localized names for various measurement systems.
+
+        >>> Locale('fr', 'FR').measurement_systems['US']
+        u'am\\xe9ricain'
+        >>> Locale('en', 'US').measurement_systems['US']
+        u'US'
+
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
```

Comment:
This would have been a great use of a Python 3.4 Enum. Just an observation, no need to do anything here. :)